examples/classification/test_MultinomialCNB.py

```diff
@@ -0,0 +1,123 @@
+"""
```

Comment:
This filename is very non-standard. It should be `plot_` something, and ideally, you should plot something!